src/components/platformIcon.tsx

```diff
@@ -273,6 +273,8 @@ import VueSVGLarge from 'platformicons/svg_80x80/vue.svg';
 import WasmSVGLarge from 'platformicons/svg_80x80/wasm.svg';
 import WindowsSVGLarge from 'platformicons/svg_80x80/windows.svg';
 import XamarinSVGLarge from 'platformicons/svg_80x80/xamarin.svg';
+// TODO(cg): move to platformicons once available
+import ReactRouterSVGLarge from 'public/icon/rr.svg';
```

Comment:
could we just update platformicons first before merging in this PR? I'd rather avoid this temporary code, and I think these docs can wait the extra 2-3 days.

Reply:
sure, I'll ping them again